modules/ROOT/pages/api-analytics-dashboard.adoc

```diff
@@ -26,9 +26,15 @@ image::api-analytics-detail-panel.png[]
 +
 . If you want to view API analytics in API Manager, click the *View API in API Manager* link.
 
+[[enhanced-api-metrics]]
 == Enhanced Metrics View for API Analytics Dashboards
 
-API analytics dashboards include various pages to view different types of charts:
+API analytics dashboards include various pages to view different types of charts.
+
+Titanium users who use the enhanced metrics view for API analytics can use the Anypoint Monitoring Archive API to download metric data that appears in the charts.
```

Comment:
How about this:
Titanium users who use the enhanced API analytics dashboards can also access the data directly via the Anypoint Monitoring Archive API. With this powerful API, you can navigate and discover your API metrics data using a directory hierarchy. The API allows download of files that contains JSON objects which represent........